src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
for readability I think it would be more helpful to pull the callback out, name it and pass it explicitly into the `Queue` method here. Passing `info[1]` doesn't really tell us much from an initial glimpse.